draft-ietf-taps-interface.md

```diff
@@ -1378,6 +1378,10 @@ It can be used to identify send events (see {{send-events}}) related to a specif
 The optional endOfMessage parameter supports partial sending and is described in
 {{send-partial}}.
 
+Framers can be used to instruct the transport service to pre- or suspend the message data with
+additional information that can be used at the reeceiver to detect messages bourndaries. This
+is further decribed in {{framing}}.
```

Comment:
"pre- or suspend" doesn't seem right. I think you mean to say "pre- or append" - and then it would have to be "pre- or append additional information to the message data" rather than "message data with...". Also, I'm not sure it's correct to say that a "transport service" is instructed to do something? Finally, there are typos ("reeceiver" and "bourndaries").

I suggest to replace the first sentence with:
"Framers can be used to instruct the transport system to extend the message data with additional information that can be used at the receiver to detect message boundaries.".

Reply:
Done

Reply:
Actually "instruct" is probably really not the right term here..

Reply:
Thanks. And I think "instruct" is ok (I don't have a better idea)